.github/workflows/linters.yml

```
@@ -42,3 +42,8 @@ jobs:
 
       - name: Test with pytest
         run: pytest
+
+      - name: codecov-umbrella
```

```suggestion
      - name: Upload test results to Codecov
```
just to be more readable